cpp/src/aztec/honk/pcs/commitment_key.hpp

```diff
@@ -80,7 +80,6 @@ class CommitmentKey {
             const_cast<Fr*>(polynomial.data()), srs.get_lagrange_points(), degree, pippenger_runtime_state);
     };
 
-  private:
```

Comment:
This will require instantiating your IPA tests not with `CommitmentSchemeParams` (probably this should go away and we should instantiate the typed tests in each file with appropriate parameters) but with whatever new parameter struct you create.

Reply:
Thanks for the suggestion. I have reverted the change inside `kzg` namespace. Instead, I have added separate `commitment_key` and `verification_key` classes inside `ipa` namespace. Here I have kept `srs` and `pippenger_runtime_state` as public member variables. This seems reasonable to me because we are actively using these member variables inside `reduce_prove` and `reduce_verify` methods in `ipa` unlike other commitment schemes. Please let me know if you have any thoughts on this.